### DISCUSSION TITLE
Welcome to wechatDownload Discussions!

### DISCUSSION THREAD

**Discussion by qiye45** [qiye45/wechatDownload#75] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xu8117** [comment]
Will it be open source?

**Comment by qiye45** [comment]
no plans lately,but if I know you, I can invite you to a private repository.

**Reply by xu8117** [reply]
If possible, can you tell me what technology stack is used?

**Reply by qiye45** [reply]
use qt

**Comment by yydsljk** [comment]
![微信图片_20241125162735](https://github.com/user-attachments/assets/dfdbf621-63b4-4f95-af04-6a673e0731cc)
![微信图片_20241125162826](https://github.com/user-attachments/assets/eeeb6339-88a9-4edb-b762-a2d411086cb9)
为什么微信打开复制的链接，网页显示请在微信客户端打开链接~一直获取秘钥失败。请大佬告知一下

**Reply by qiye45** [reply]
看看这个 https://github.com/qiye45/wechatDownload/issues/9

**Reply by bigdan05** [reply]
3.4版本无法保存word

**Comment by iris-1913** [comment]
大佬求问mac版本只有2.6吗 这个版本没法获取公众号id

**Comment by dengjincheng1943-collab** [comment]
已经不能通过生成的「公众号主页链接」进入对应【公众号主页】了...还会更新吗？有沟通群吗

**Reply by qiye45** [reply]
聊天群：https://t.me/changfengbox
